www-apps/grafana-bin/Manifest

```diff
@@ -1,2 +1 @@
-DIST grafana-bin-5.1.3.tar.gz 52436314 BLAKE2B e5ff613a6d65018b6c205a88a520b5f07ea32955cb8f4b75d1eef05946d93c50272edecf80c8a4a6b8b4a70fbae7c90c71f1b833a31ff56ed71c59dda3007b4c SHA512 3d0fcc2015d95a5334d571f4ea61fa1bb7ead1b4131342e3bf167c832c0f951ae6669049da73a8d954117d54b5422be9203505db19b370ec337c481900d7fea5
```

Comment:
make your commits atomic first, one for removal and one for addition

Reply:
Alright, thanks for the feedback! First I did it like that, but then I figured one change would not make sense without the other and squashed them. Will split and push again in a bit.

Reply:
@Zlogene I reverted my squash.